src/libs/histogram.c

```diff
@@ -149,17 +160,29 @@ const char **views(dt_lib_module_t *self)
 
 uint32_t container(dt_lib_module_t *self)
 {
-  return DT_UI_CONTAINER_PANEL_RIGHT_TOP;
+  dt_lib_histogram_position_t position = DT_LIB_HISTOGRAM_RIGHT;
```

Comment:
better adding `plugins/darkroom/histogram/position` in darktableconfig.xml.in and set default value there (1). Then you don't need to check for the existence of this key here.

Reply:
done